indico/modules/users/forms.py

```diff
@@ -83,3 +83,16 @@ class MergeForm(IndicoForm):
                                  description=_('The user that will be merged into the target one'))
     target_user = PrincipalField(_('Target user'), [DataRequired()], serializable=False,
                                  description=_('The user that will remain active in the end'))
+
+
+class UserManagementForm(IndicoForm):
+    notify_on_new_account = BooleanField(_('Notify when new account is created'), widget=SwitchWidget(),
+                                            description=_('Enable sending emails to admins when a new account is created'))
+    local_account_creation = BooleanField(_('Enable local registration'), widget=SwitchWidget(),
+                                            description=_('Enable users to register locally'))
+    account_moderation_workflow = BooleanField(_('Account moderation'), widget=SwitchWidget(),
+                                                description=_('Enable account moderation workflow'))
+
+    def is_submitted(self):
+        from flask import request
```

Comment:
Having imports global is the proper way to do it unless they have to be local to avoid circular imports. Python has a module scope so you don't pollute a truly global namespace like you would do e.g. in PHP.